modules/rostests/apitests/shell32/ExtractIconEx.cpp

```diff
@@ -22,16 +22,33 @@ EXTRACTICONTESTS IconTests[] =
     {L"%SystemRoot%\\System32\\autochk.exe", 0},
 
     /* Non-existing files */
-    {L"%SystemRoot%\\non-existent-file.sdf", 0}
+    {L"%SystemRoot%\\non-existent-file.sdf", 0},
+
+    /* Multiple icons in the same ICO file (6 icons) */
+    {L"%SystemRoot%\\bin\\sysicon.ico", 1},
+
+    /* Multiple icons in the same EXE file (18 icons) */
+    {L"%SystemRoot%\\explorer.exe", 18},
+
+    /* One standard icon and one PNG in same file */
+    {L"%SystemRoot%\\bin\\ROS.ico", 1}
```

Comment:
Can you embed `ROS.ico` and `sysicon.ico` into the resource?
Are the `explorer.exe` tests compatible to WinXP/Win2k3?

Reply:
https://github.com/reactos/reactos/blob/9686771fc07fd9bb9c77344f88c5fc1f863f5fe3/modules/rostests/apitests/shell32/resource.rc#L7

Reply:
```txt
2 RCDATA "ROS.ico"
3 RCDATA "sysicon.ico"
```

Reply:
```c 
HRSRC hRes = FindResourceW(NULL, MAKEINTRESOURCEW(2), RT_RCDATA);
HGLOBAL hData = LoadResource(NULL, hRes);
...
```

Reply:
@katahiromz, I have looked into how to do this and I am not able to figure it out for now. Maybe you can offer a suggestion or maybe just leave it for the next PR? Thanks.

Reply:
> ```c
> HRSRC hRes = FindResourceW(NULL, MAKEINTRESOURCEW(2), RT_RCDATA);
> HGLOBAL hData = LoadResource(NULL, hRes);
> ...
> fclose(fout);
> ```

Huh just use LoadIcon or LoadImage ...

Reply:
OK, @katahiromz and @HBelusca, I believe that I now have the icons embedded in the EXE file as was suggested. Please review again and offer your comments. Thanks.